parser/ast/ddl.go

```diff
@@ -2560,6 +2560,8 @@ const (
 	AlterTableOrderByColumns
 	// AlterTableSetTiFlashReplica uses to set the table TiFlash replica.
 	AlterTableSetTiFlashReplica
+	// AlterTableSetTiFlashMode uses to alter the table mode of TiFlash.
+	AlterTableSetTiFlashMode
```

Comment:
I'm not sure whether adding a new enum at the middle will cause compatibility problems. @crazycs520 PTAL, thanks

Reply:
Missed that, yes, it is a problem. @hongyunyan, please change.